src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -1,8 +1,45 @@
 -- Enable RCSI
-ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON;
+ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON
+GO
 
 -- Create Resource table
-IF NOT EXISTS ( SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'Resources' )
+IF NOT EXISTS ( SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'SchemaVersion' )
 BEGIN
-    CREATE TABLE Resources (ResourceId int PRIMARY KEY);
+    CREATE TABLE SchemaVersion (
+        [Version] int PRIMARY KEY, 
+        [Status] varchar(10) 
+    )
 END
+GO
+
+CREATE PROCEDURE dbo.SelectCurrentSchemaVersion
+AS BEGIN
+    SELECT
+        MAX([Version])
+    FROM SchemaVersion 
+    WHERE [Status] = 'complete'
+END
+GO
+
+CREATE PROCEDURE dbo.UpsertSchemaVersion(
+        @Version int,
```

Comment:
I suggest we adopt a set of naming conventions for SQL, such as this one: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
Following that, variable names would be camelCased